source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -389,6 +389,14 @@ namespace
                             const RtToken inputType(elem->getType());
                             RtInput input = schema.createInput(socketName, inputType);
                             socket = schema.getInputSocket(input.getName());
+
+                            // Set the input value
+                            const string& valueStr = elem->getValueString();
```

Comment:
Need to copy the value over now when the inputsocket is created as we don't read it in before-hand.